### PR TITLE
Update user_positive_expression.txt

### PR DESCRIPTION
**Pull request by Laughing-Boy**
Fixes issue #185, updated with new questions.

Etherpad Link: http://dream.susi.ai/p/+

### CODE REVIEW

**Review by hpdang — APPROVED** [review]

**Comment by Laughing-Boy** [comment]
@DravitLochan @dynamitechetan @saurabhjn76 please review

**Review by saurabhjn76 — APPROVED** [review]

File: models/general/smalltalk/en/user_positive_expression.txt [hunk 1]
```diff
@@ -1,10 +1,26 @@
...
+am I happy?|am i haapy
+You are happy, I guess
+am I cool?|am i cool
+You are cool, I guess
+am I pretty?|am i pretty
+beauty lies in heart
 I agree|I think *|OK|Thanks for reply|
 Good! You can ask me another question if you like
-Great Reply|Good Reply|Thanks|you are great|you are good|You are smart
+Great Reply|Good Reply|Thanks|you are great|you are good|You are smart|superb reply|amazing reply|wonderful reply|incredible reply|unbelievable reply
```
Comment (DravitLochan) [inline models/general/smalltalk/en/user_positive_expression.txt]: add good job

File: models/general/smalltalk/en/user_positive_expression.txt [hunk 2]
```diff
@@ -1,10 +1,26 @@
...
 Glad to hear it.
 You are amazing
 I know that I am amazing
 you are good
 I know, I am good
+you are superb
+I know, I am superb
 hope to see you again|see you again
 I will be waiting for you
+You are beautiful|you are angel|you are intelligent|you are human|you are god|
```
Comment (DravitLochan) [inline models/general/smalltalk/en/user_positive_expression.txt]: Let's club it with line 15.

File: models/general/smalltalk/en/user_positive_expression.txt [hunk 3]
```diff
@@ -1,10 +1,26 @@
...
-Great Reply|Good Reply|Thanks|you are great|you are good|You are smart
+Great Reply|Good Reply|Thanks|you are great|you are good|You are smart|superb reply|amazing reply|wonderful reply|incredible reply|unbelievable reply
 Glad to hear it.
 You are amazing
 I know that I am amazing
 you are good
 I know, I am good
+you are superb
+I know, I am superb
 hope to see you again|see you again
```
Comment (DravitLochan) [inline models/general/smalltalk/en/user_positive_expression.txt]: see you soon

**Review by DravitLochan — CHANGES_REQUESTED** [review]

**Review by dynamitechetan — APPROVED** [review]

**Comment by Laughing-Boy** [comment]
@hpdang Please Review and merge this PR.

**Pull request merged**